blocks/bricks/base.py

```diff
@@ -512,7 +505,7 @@ class Brick(Annotation):
 
     Examples
     --------
-    By default, bricks have lazy initialization enabled.
+    Most bricks have lazy initialization enabled.
```

Comment:
Nope, a user can technically write his own brick without the `@lazy` decorator. I think this should be allowed (and I actually edited the code specifically to do so) because it can make for easier development when quickly hacking a brick together e.g. you can just do
```python
StupidBrick(Brick):
    def __init__(self, in, out):
        self.parameters.append(theano.shared(np.random.rand((in, out))))

    @application
    def apply(self, x):
        return x + self.parameters[0]
```